### ISSUE TITLE
Zablokování tisku mezičasů pokud proběhne vyčítání při otevřeném dialogu

### ISSUE THREAD

**Issue by janpalas** [Quick-Event/quickbox#663]
Pokud mám otevřený nějaký dialog (např.) "Přiřadit čip závodníkovi" a v té chvíli někdo provede vyčítání čipu, tak se zablokuje tisk mezičasů (tzn. vyčítání funguje, ale lístečky se automaticky netisknou - lze tisknout ručně pomocí "Vytisknout mezičasy"). Pomůže pouze restart programu nebo (a to jsme už nestihli dostatečně otestovat) odškrtnout a znovu zaškrtnout "Automatický tisk".

- Tuto chybu jsme opakovaně pozorovali na ČPŠ 25.-26.9.2021.
- Pokud stihnu dialog zavřít, než někdo vyčte, k zablokování tisku nedojde. 
- Přikládám zálohu ze závodu.

**Comment by paukert** [comment]
To vypadá podobně jako #662

**Comment by janpalas** [comment]
> To vypadá podobně jako #662

Ano, předpokládám že #662 vytvořil @fvacek na základě našeho rozhovoru o víkendu na závodech. Nicméně mi přišlo, že název popisuje něco jiného, proto jsem vytvořil toto issue.

**Comment by paukert** [comment]
Zkusil jsem chybu na aktuální dev verzi QE na W10 reprodukovat, ale nepodařilo se mi to. Při otevřeném dialogu pro přiřazení čipu nebo dialogu pro přidání závodníka se mi lístečky normálně tiskly dále. Pravděpodobně to tedy bylo způsobeno ještě nějakým dalším faktorem.

**Comment by fvacek** [comment]
Muzu poprosit @janpalas , aby se to pokusil zreprodukovat s prilozenym QBE stejen jako @paukert. Budeme to muset orezavat tak dlouho, az se to podari zreprodukovat. Treba se to deje jenom s PSQL a ne s SQLite.
